src/Symfony/Bundle/FrameworkBundle/Routing/Attribute/AsRoutingConditionService.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Routing\Attribute;
+
+#[\Attribute(\Attribute::TARGET_CLASS)]
+class AsRoutingConditionService
```

Comment:
could you add a small desription and example, like you have already made in the PR, explaining that you can use the service in the routing condition like so `#[Route('/page', condition: "service('foo').bar()")]`

Reply:
done (and rebased)